rspec-mode.el

```diff
@@ -755,6 +755,13 @@ or a cons (FILE . LINE), to run one example."
         (concat (rspec-project-root) "spec/spec_helper.rb"))
        (re-search-forward "include +FactoryGirl::Syntax::Methods" nil t)))))
 
+(defun rspec-expose-dsl-globally-p ()
+  "Check whether expose_dsl_globally is set to false in spec_helper."
+  (with-temp-buffer
+    (insert-file-contents
+     (concat (rspec-project-root) "spec/spec_helper.rb"))
```

Comment:
The usual idiom is to use `expand-file-name` and pass the directory as the second argument.